util/urql/init-client.js

```diff
@@ -24,8 +25,15 @@ export default function initUrqlClient(initialState, token) {
     urqlClient = createClient({
       url: "https://api.github.com/graphql",
       // Active suspense mode on the server-side
-      suspense: isServer,
-      exchanges: [dedupExchange, cacheExchange, ssrCache, fetchExchange],
+      suspense: true,
+      exchanges: [
+        !isServer && devtoolsExchange,
+        dedupExchange,
+        suspenseExchange,
+        cacheExchange,
+        ssrCache,
```

Comment:
Found the bug 😁 The `ssrExchange` behaves differently on the client and server. Previously we haven't taken the `suspenseExchange` into account, so we simply assume `!!client.suspense` means we're on the server and `!client.suspense` means we're on the client.

So, what happens is, since you're using SSR + `@urql/exchange-suspense` the client-side is always assuming that it's actually on the server-side. So it never _deletes_ its cached SSR data.

We'll have to fix this by maybe adding `isServer: boolean` to the `ssrExchange({ ... })` options.

Reply:
Fix: https://github.com/FormidableLabs/urql/pull/369

Reply:
it fixed an issue

Reply:
thank you

Reply:
there are no errors, and update works. but I still can't make Suspense's fallback to be triggered even with 5s delay network throttling
<img width="501" alt="Screen Shot 2019-08-06 at 2 39 43 AM" src="https://user-images.githubusercontent.com/559321/62504094-51482b00-b7f7-11e9-9ad4-9529865f9baf.png">